<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/charts2.png</normaloff>:/images/images/charts2.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLogGraph">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>24</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>Show Log Graph</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dblSpinBox">
    <property name="geometry">
     <rect>
      <x>787</x>
      <y>18</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetColor">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>75</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="text">
     <string>Change Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCustomGraph">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>118</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>Show Linear graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowTestChart">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>160</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>Show test Chart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangeDate">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>202</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="text">
     <string>add years</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadFile">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>279</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTutorial">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>358</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>Tutorial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenFolders">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>400</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="text">
     <string>Open Folders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowTable">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>441</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>126</width>
      <height>26</height>
     </size>
    </property>
    <property name="text">
     <string>Show Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>482</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>127</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelColorTest">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>18</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>color Test Label</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstFolders">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>243</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineColor">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>73</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="newLineColor">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>73</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>203</y>
      <width>389</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineDate"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineYears"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineDateResult"/>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="spinBox99">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>46</width>
      <height>27</height>
     </rect>
    </property>
    <property name="suffix">
     <string extracomment="99"/>
    </property>
    <property name="prefix">
     <string extracomment="99"/>
    </property>
   </widget>
   <widget class="QComboBox" name="combo99">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>79</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line99">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo99_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>79</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo99_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>79</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo99_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>79</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo99_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>79</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
